content/blog/2018-08-28-birds-ocr.md

```diff
@@ -0,0 +1,411 @@
+of Australian birds, potentially leveraging [the `taxizedb` package by
+Scott Chamberlain](https://github.com/ropensci/taxizedb), and to then
+fuzzy-match them with the names we have.
+
+``` r
+unique(bird_names$gnr)
+```
+
+    ## [[1]]
+    ## # A tibble: 1 x 5
+    ##   user_supplied_name submitted_name  matched_name   data_source_tit~ score
+    ## * <chr>              <chr>           <chr>          <chr>            <dbl>
+    ## 1 climacteris picum~ Climacteris pi~ Climacteris p~ NCBI             0.988
+    ## 
+    ## [[2]]
+    ## [1] NA
+    ## 
+    ## [[3]]
+    ## NULL
+    ## 
+    ## [[4]]
+    ## # A tibble: 1 x 5
+    ##   user_supplied_na~ submitted_name  matched_name  data_source_title  score
+    ## * <chr>             <chr>           <chr>         <chr>              <dbl>
+    ## 1 austrodicaeum ii~ Austrodicaeum ~ Austrodicaeu~ The Interim Regis~  0.75
+    ## 
+    ## [[5]]
+    ## # A tibble: 1 x 5
+    ##   user_supplied_name submitted_name  matched_name   data_source_tit~ score
+    ## * <chr>              <chr>           <chr>          <chr>            <dbl>
+    ## 1 melithreptus laet~ Melithreptus l~ Melithreptus ~ CU*STAR          0.988
+    ## 
+    ## [[6]]
+    ## # A tibble: 1 x 5
+    ##   user_supplied_na~ submitted_name matched_name  data_source_title   score
+    ## * <chr>             <chr>          <chr>         <chr>               <dbl>
+    ## 1 rad isdlvorniode  Rad isdlvorni~ Rad Baker & ~ The Interim Regist~  0.75
+
+### Conclusion
+
+#### rOpenSci packages supporting this (and your) workflow
+
+In this post, we made use of R packages quite useful to wrangle
+information from diverse formats:
+
+-   [`magick`](https://github.com/ropensci/magick) for image
+    manipulation,
+
+-   [`tesseract`](https://github.com/ropensci/tesseract) for optical
+    character recognition,
+
+-   [`cld2`](https://github.com/ropensci/cld2)/[`cld3`](https://github.com/ropensci/cld3)
+    for language detection.
+
+We also used [a function from
+`taxize`](https://github.com/ropensci/taxize) allowing to use [the
+Global Name Resolver](https://resolver.globalnames.org/about). Discover
+more packages from our suite [here](https://ropensci.org/packages/).
+
+#### Applicability of this OCR bird-naming workflow
+
+Actually, the Biodiversity Heritage Library itself provides OCR output
+for its collection, see [this
+example](https://www.biodiversitylibrary.org/item/250938#page/37/mode/1up).
+I wasn’t able to find information about the software powering this OCR.
+What I *was* able to find out is that the BHL uses [purposeful
+gaming](https://biodivlib.wikispaces.com/Purposeful+Gaming) in [its OCR
+workflow](https://biodivlib.wikispaces.com/file/view/Workflowdiagram.pdf).
+The raw OCR results aren’t much better than what we got in this post
+which is comforting.
+
+#### More data from the Biodiversity Heritage Library
+
+If you’re interested in the data, images but not only, of the
```

Comment:
suggested rephrase:

If you’re interested in other types of data from the BHL, in addition to the images,